src/localize-router.parser.ts

```diff
@@ -110,7 +106,12 @@ export abstract class LocalizeParser {
 
     /** translate routes */
     const res = this.translateRoutes(selectedLanguage);
-    return res.toPromise();
+    return new Promise((resolve, reject) => {
```

Comment:
You don't need to create Promise here. Use `toPromise()` method on Observable.

Reply:
@imsingh As far as I read, `toPromise()` was part of rxjs/operators version 5.5 and has been removed on v6 (see this thread https://github.com/ReactiveX/rxjs/issues/3482).

What do you think is the best option we have?

Reply:
@NKjoep yeah it was removed as lettable operator. But it's now method on Observable class(same like previously). Take a look at https://github.com/ReactiveX/rxjs/issues/2868